src/Controller/Api/FunderController.php

```diff
@@ -19,12 +19,14 @@ class FunderController extends AbstractController
      *
      * @param  Request          $request          A request object.
      * @param  FunderRepository $funderRepository The FunderRepository class.
-     * @return Response                           A http response object that sends JSON to browser.
+     *
+     * @return Response A http response object that sends JSON to browser.
      */
     #[Route('/api/funders', name: 'app_api_funders_by_name')]
     public function getFunderByName(Request $request, FunderRepository $funderRepository): Response
     {
         $userQueryString = $request->query->get('queryString');
+        $userQueryString = null;
```

Comment:
Is this debugging code?

Reply:
yes :-(

In the function that is called, it used the escape method (double quotes) to from a string, I just wanted to know how it would behave with a null. And it just returns `"%%"` which is a string. But normally allowing a null in this function would not be ok if the function wanted a string. `setParameter` 2nd parameter allows for `mixed` but not null.
So this is correct, but you have to be careful if that parameter criteria might change. 
One thing to make sure there are no issue is the add `$userQueryString ?? ''` in `findFunderByPartialName`